lib/gap_intelligence/client/merchant_pricing_trends.rb

```diff
@@ -0,0 +1,18 @@
+module GapIntelligence
+  # @see http://api.gapintelligence.com/api/doc/v1/merchant_pricing_trends.html
+  module MerchantPricingTrends
+    # Requests a list of merchant pricing trends
+    #
+    # @param params [Hash] parameters of the http request
+    # @param options [Hash] the options to make the request with
+    # @yield [req] The Faraday request
+    # @return [MerchantPricingTrend] the requested data
+    # @return [RequestError] error messages
+    # @see http://api.gapintelligence.com/api/doc/v1/merchant_pricing_trends/index.html
+    def merchant_pricing_trends(params = {}, options = {}, &block)
+      default_option(options, :record_class, MerchantPricingTrend)
+      default_option(options, :response_processor, Record)
```

Comment:
@shavkat-samatov, seems like `response_processor` is not used anywhere and can be removed.